cli/tests/pcluster/config/test_utils.py

```diff
@@ -34,3 +36,35 @@ def test_get_instance_vcpus(boto3_stubber, valid_instance_type, expected_vcpus):
 
     boto3_stubber("ec2", mocked_requests)
     assert_that(get_instance_vcpus(instance_type)).is_equal_to(expected_vcpus)
+
+
+@pytest.mark.parametrize(
+    "region, free_tier_instance_type, default_instance_type",
+    [
+        ("us-east-1", "t2.micro", "t2.micro"),
+        ("eu-north-1", "t3.micro", "t3.micro"),
```

Comment:
Could we repeat one set of dimensions to check the internal cache? On the second call there should be no boto3 calls triggered.